Sources/SwiftDocC/Semantics/Redirect.swift

```diff
@@ -31,7 +35,7 @@ public final class Redirect: Semantic, AutomaticDirectiveConvertible {
         "oldPath" : \Redirect._oldPath,
     ]
     
-    static var hiddenFromDocumentation = true
+    static var hiddenFromDocumentation = false
```

Comment:
@patshaughnessy I'm not following here, why is this needed?

Reply:
We use the `generate-symbol-graph` command to document all of DocC's directives automatically. This command generates a symbol graph file for these, which is then used when we compile and publish documentation for DocC itself. This particular setting is [used here](https://github.com/apple/swift-docc/blob/main/Sources/generate-symbol-graph/main.swift#L182) to filter out some directives. (I'm not sure why we need to filter them at all, to be honest). Since `hiddenFromDocumentation` was set to true previously for `@Redirected`, I changed it to false to include docs for this again.